rs/ledger_suite/icp/src/lib.rs

```diff
@@ -1096,6 +1097,12 @@ pub struct GetBlocksArgs {
     pub length: u64,
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize, Serialize)]
+pub struct ICRC3GetBlocksArgs {
```

Comment:
Isn't camel case like this?


```suggestion
pub struct Icrc3GetBlocksArgs {
```

Or is there a local convention here?

Reply:
I think because ICRC is probably an abbreviation, it is okay to keep its letters all Capital in camel case. Just let me know, if the convention in the company is different.